css/module.css

```diff
@@ -0,0 +1,103 @@
+/*BANNER MODULE*/
+
+.banner li {
+  display: inline-block;
+  font-size: 20px;
+}
+
+.banner li:nth-of-type(6) {
+  margin-right: 10px;
+}
+
+.banner__logo h1 {
+  font-family: 'Yellowtail', cursive;
+  font-size: 50px;
+}
+
+.banner__logo h2 {
+  font-family: 'Abel', sans-serif;
+  font-size: 10px;
+  color: gray;
+}
+
+/*RECIPE HEADER MODULE*/
+
+.recipe__header {
+  margin-top: 35px;
+  border-top: 15px solid;
+  border-image: url(https://files.slack.com/files-pri/T039KG69K-F04B91DDD/dash-border.png) 35 23 27 26 fill repeat stretch;
+}
+
+.recipe__header h1 {
+  font-size: 42px;
+  font-family: 'Oswald', sans-serif;
+}
+
+.recipe__header p {
+  font-family: 'Roboto', sans-serif;
+  font-size: 14px;
+  color: gray;
+  margin-top: 0px;
+}
+
+.recipe__header__print {
+  margin-top: 55px;
```

Comment:
This is inside a flex container, right? If so, you can likely put in align-self: baseline